sdk/communication/Azure.Communication.Chat/tests/Infrastructure/ChatLiveTestBase.cs

```diff
@@ -9,7 +9,7 @@ namespace Azure.Communication.Chat.Tests
 {
     public class ChatLiveTestBase : RecordedTestBase<ChatTestEnvironment>
     {
-        public ChatLiveTestBase(bool isAsync) : base(isAsync)
+        public ChatLiveTestBase(bool isAsync) : base(isAsync,RecordedTestMode.Playback)
```

Comment:
we should not hardcode to playback or it cannot run in live mode

Reply:
removed it, was mistakenly committed before